dev/Pager/PrototypePager/Themes/PrototypePager.xaml

```diff
@@ -0,0 +1,181 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:MUXControlsTestApp"
+    xmlns:controls="using:Microsoft.UI.Xaml.Controls"
+    xmlns:local1="using:MUXControlsTestApp.Scripts">
+
+
+    <local1:Int32ToDoubleConverter x:Key="IntToDouble"/>
+    <local1:IntToStrConverter x:Key="IntToStr"/>
+    <local1:Int32MinusOneConverter x:Key="IntMinusOne"/>
+
+    <DataTemplate x:Key="IntToButton">
+        <Button Content="{Binding}" MinWidth="35"
```

Comment:
>35 [](start = 46, length = 2)

This should be a factored out into a resource

Reply:
Done!